.env

```diff
@@ -0,0 +1,45 @@
+# In all environments, the following files are loaded if they exist,
+# the latter taking precedence over the former:
+#
+#  * .env                contains default values for the environment variables needed by the app
+#  * .env.local          uncommitted file with local overrides
+#  * .env.$APP_ENV       committed environment-specific defaults
+#  * .env.$APP_ENV.local uncommitted environment-specific overrides
+#
+# Real environment variables win over .env files.
+#
+# DO NOT DEFINE PRODUCTION SECRETS IN THIS FILE NOR IN ANY OTHER COMMITTED FILES.
+#
+# Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_SECRET=2l386svw9n673j2uj52mqcxr74qj5jnd
+#TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
+#TRUSTED_HOSTS='^localhost|example\.com$'
+###< symfony/framework-bundle ##
+
+###> doctrine/doctrine-bundle ###
+# Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
+DATABASE_URL=pgsql://postgres:123@postgres:5432/ergonode?serverVersion=9.7&charset=utf8
+###< doctrine/doctrine-bundle ###
+
+###> lexik/jwt-authentication-bundle ###
+JWT_PRIVATE_KEY_PATH=config/jwt/private.pem
+JWT_PUBLIC_KEY_PATH=config/jwt/public.pem
+JWT_PASSPHRASE=1234
+JWT_TOKEN_TTL=86400
+###< lexik/jwt-authentication-bundle ###
+
+###> nelmio/cors-bundle ###
+CORS_ALLOW_ORIGIN=^https?://(localhost|127\.0\.0\.1)(:[0-9]+)?$
+###< nelmio/cors-bundle ###
+
+###> app settings ###
+APP_HOST=localhost
+APP_SCHEME=http
+APP_URL=http://localhost:8001
```

Comment:
Why 8001 ?? Symofny server default work on 8000 and should stay that port

Reply:
Maybe it's your default port but even before https://github.com/ergonode/docker/pull/1
it is was : 
`command: 'bash -c "cd /var/www && composer install -o && bin/console -v s:r 0.0.0.0:8001"`
in file https://github.com/ergonode/docker/blob/master/docker-compose.yml